pom.xml

```diff
@@ -80,7 +80,7 @@
             <!--Overriding spring-web version to remove vulnerability-->
             <groupId>org.springframework</groupId>
             <artifactId>spring-web</artifactId>
-            <version>6.1.6</version>
+            <version>6.1.10</version>
```

Comment:
Have you checked if you actually need to override spring web version anymore? Possibly the one that comes with spring boot 3.3.2 does not have the vulnerability so you could safely remove this?

Reply:
this has been removed and resolved